traffic_portal/app/src/common/service/application/ApplicationService.js

```diff
@@ -38,6 +38,10 @@ var ApplicationService = function($rootScope, $anchorScroll, $http, messageModel
 
         // jquery DataTables default overrides
         $.extend(true, $.fn.dataTable.defaults, {
+            "dom": "lfBrtip",
+            "buttons": [
+                { "extend": "csv", "className": "btn-link", "titleAttr": "Export to CSV" }
+            ],
```

Comment:
Is there any way this can be changed to a download icon like <kbd>⭳</kbd> or something? Failing that, "Export as CSV" or simply "Export" would be more clear than just "CSV", IMO.

Reply:
i did try an icon and it looked kinda weird just hanging out on the top of the table. But yes, "Export as CSV" would be better. clarity is good.

Reply:
done